README.md

```diff
@@ -9,8 +9,8 @@ Installs and configures Dan Bernstein's DNS tinydns, aka djbdns. Services are co
 The following platforms are supported via test kitchen.
 
 * Ubuntu 10.04, 12.04, 14.04
-* Debian 7.6
-* CentOS 6.5, 5.10
+* Debian 7.8
```

Comment:
What no Debian 8? :smile_cat: 

(I don't think we have bento boxes for that yet)